lib/components/settings_manager.dart

```diff
@@ -20,17 +20,25 @@ class PrefKeys {
   static const eventsShowRed = 'events_show_red';
   static const eventsShowBlue = 'events_show_blue';
   static const eventsShowGreen = 'events_show_green';
+  static const eventsNotifyNA = 'events_notify_na';
+  static const eventsNotifyJP = 'events_notify_jp';
+  static const eventsNotifyKR = 'events_notify_kr';
 
   static const uiTheme = 'ui_theme';
   static const uiDarkMode = 'ui_dark_mode';
 
   static const hideUnreleasedMonsters = 'hide_unreleased_monsters';
+
+  static const trackedDungeons = 'tracked_dungeons';
+
+  static const inDevMode = 'in_dev_mode';
```

Comment:
This works fine but it seems a bit unnecessary. I don't think it needs to be stored as a pref. If you want you can just add a static toggle here and set it from main directly.

Reply:
Yeah now that you mention it, having as a pref is not good. Brainfarted and forgot you can just import that variable anywhere since it's top level in main